app/graph/types/explainer_type.rb

```diff
@@ -0,0 +1,21 @@
+class ExplainerType < DefaultObject
+  description "Explainer type"
+
+  implements GraphQL::Types::Relay::Node
+
+  field :dbid, GraphQL::Types::Int, null: true
+  field :title, GraphQL::Types::String, null: true
+  field :description, GraphQL::Types::String, null: true
+  field :url, GraphQL::Types::String, null: true
+  field :language, GraphQL::Types::String, null: true
+  field :user_id, GraphQL::Types::Int, null: true
+  field :team_id, GraphQL::Types::Int, null: true
+  field :user, UserType, null: true
+  field :team, TeamType, null: true
+
+  field :tags, TagType.connection_type, null: true
+
+  def tags
+    object.get_annotations('tag').map(&:load)
```

Comment:
Change to be an `ActiveRecord::Relation` (e.g., `Tag.where(annotated_type: ...)`)